keyboards/handwired/drevogramr_qmk/config.h

```diff
@@ -0,0 +1,181 @@
+/* Copyright 2021 wwwMADwww <4d4144h@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xD2E0
+#define PRODUCT_ID      0x62A2
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Handwire
+#define PRODUCT         Drevo Gramr QMK
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 16
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+*/
+
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+// does not matter in custom matrix scanning mode, just for information
+#define DIODE_DIRECTION ROW2COL
+
+
+#define LAYER_STATE_8BIT
```

Comment:
Is there a specific reason to limit the layers here?

Reply:
No, removed.